src/lib/utils.js

```diff
@@ -1,3 +1,16 @@
+/**
+ * @typedef {import("../components/common.types.js").ExpirationWarningThresholds} ExpirationWarningThresholds
+ */
+
+/** @type {ExpirationWarningThresholds} */
+const DEFAULT_THRESHOLDS = [
```

Comment:
Maybe I should put this inside the `isExpirationClose` helper instead? :shrug: 
I placed it here because I wanted to use it as the function param default value but we can easily refactor to fallback to this is no thresholds are provided :+1: